architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -137,11 +137,22 @@ Pipeline workflows are defined in a Jenkinsfile,
 either embedded directly in the build configuration,
 or supplied in a Git repository and referenced by the build configuration.
 
-The first time a project defines a build configuration using a Pipeline strategy,
-{product-title} instantiates a Jenkins server to execute the pipeline.
-Subsequent Pipeline build configurations
-in the project share this Jenkins server.  For more details on how the Jenkins
-server is deployed, see xref:../../install_config/configuring_pipeline_execution.adoc#overview[configuring pipeline execution].
+The first time a project defines a build configuration using a Pipeline
+strategy, {product-title} instantiates a Jenkins server to execute the
+pipeline.  Subsequent Pipeline build configurations in the project share this
+Jenkins server. 
+
+ifdef::openshift-online[]
+If you would like to block autoprovisioning (if you are running a Jenkins
+server external to {product-title}, for example), create a Service with the
+name `*jenkins*` in your project, and configure it to point to the external
+Jenkins.
+xref:../dev_guide/integrating_external_services.adoc#dev-guide-integrating-external-services[Learn more about integrating external services.]
```

Comment:
drop these lines (145-150).  in general we try to keep implementation/config details out of the "architecture" section.  it's a high level overview of concepts, not a usage guide.